src/main/kotlin/telegram/Menu.kt

```diff
@@ -263,53 +336,107 @@ internal fun showChosenSettingsMenu(chatId: Long, messageId: Long, bot: Bot, cho
     }
 }
 
-internal fun showKickMenu(game: Game, messageId: Long, bot: Bot, chatId: Long) {
+internal fun showChosenHostInfoSettings(chatId: Long, messageId: Long, bot: Bot, hostId: Long) {
+    val msgId = if (messageId == -1L) {
+        bot.sendMsg(chatId, "Настройки ведущего").msgId
+    } else {
+        messageId
+    }
     bot.editMessageReplyMarkup(
         ChatId.fromId(chatId),
-        messageId,
+        msgId,
         replyMarkup = inlineKeyboard {
-            button(blankCommand named "Исключенные игроки")
-            kicks.find { gameId == game.id }.forEach {
-                accounts.get(it.player)?.let { acc ->
-                    button(blankCommand named acc.fullName())
-                    button(unkickCommand, it.id, messageId)
+            hostInfos.get(hostId)?.let {
+                row {
+                    button(blankCommand named "🎮 Лимит игр")
+                    if (it.gameLimit) {
+                        button(gameLimitOnCommand named it.left.toString(), it.chatId, msgId)
+                        button(gameLimitOffCommand, it.chatId, msgId)
+                    } else {
+                        button(gameLimitOnCommand, it.chatId, msgId)
+                    }
+                }
+                row {
+                    button(blankCommand named "⏰ Срок ведения")
+                    if (it.timeLimit) {
+                        button(timeLimitOnCommand named it.until.toString(), it.chatId, msgId)
+                        button(timeLimitOffCommand, it.chatId, msgId)
+                    } else {
+                        button(timeLimitOnCommand, it.chatId, msgId)
+                    }
+                }
+                row {
+                    button(blankCommand named "👥 Передавать ведение")
+                    button(shareCommand named if (it.canShare) "On" else "Off", it.chatId, msgId)
+                }
+                row {
+                    button(blankCommand named "👇 Выбирать роли")
+                    button(canReassignCommand named if (it.canReassign) "On" else "Off", it.chatId, msgId)
+                }
+                if (admins.get(it.chatId) == null) {
+                    button(promoteHostCommand, it.chatId, msgId)
+                } else {
+                    button(blankCommand named "⚛️ Администратор")
                 }
             }
-            button(hostBackCommand, messageId)
+            button(deleteMsgCommand named "Закрыть", msgId)
```

Comment:
Why? Just why? Again, i'm sure that i've already said that in a previous review. You are sending a message id of an admin menu message, you are supposed to go back to admin menu from here, not delete the whole admin menu